app/src/controllers/project/actionCreators.js

```diff
@@ -64,6 +66,16 @@ export const updateProjectNotificationsConfigAction = (config) => ({
   payload: config,
 });
 
+export const updateProjectNotificationAction = (notification) => ({
+  type: UPDATE_NOTIFICATION,
+  payload: notification,
+});
+
+export const deleteProjectNotificationAction = (id) => ({
+  type: DELETE_NOTIFICATION,
+  payload: id,
+});
+
```

Comment:
Please move this actions down to the other new actions added in the previous tasks.
And use action name as `UPDATE_PROJECT_NOTIFICATION`, as we have system notifications that are just `*_NOTIFICATION`